### PR TITLE
ipatests: Add tests for interactive chronyd configuration

### PR DESCRIPTION
**Pull request by Tiboris**
After adding interactive configuration to installer and moved it to install_check needs to be tested.

See:https://pagure.io/freeipa/issue/7747
See:https://pagure.io/freeipa/issue/7930
Resolves: https://pagure.io/freeipa/issue/7908

### CODE REVIEW

**Comment by rcritten** [comment]
./ipatests/test_integration/test_ntp_options.py:239:80: E501 line too long (92 > 79 characters)
./ipatests/test_integration/test_ntp_options.py:240:20: E261 at least two spaces before inline comment
./ipatests/test_integration/test_ntp_options.py:247:80: E501 line too long (86 > 79 characters)

**Comment by fcami** [comment]
@Tiboris you've added one test_integration too much:  test_integration/test_integration

File: ipatests/pytest_ipa/integration/tasks.py [hunk 1]
```diff
@@ -498,8 +498,8 @@ def install_replica(master, replica, setup_ca=True, setup_dns=False,
     return result
 
 
-def install_client(master, client, extra_args=(),
-                   user=None, password=None):
+def install_client(master, client, extra_args=(), user=None,
```
Comment (miskopo) [inline ipatests/pytest_ipa/integration/tasks.py]: `extra_args` default to tuple, however, they are only used upon conversion to list as seen at `531`. Thus, this conversion is obsolete and we could benefit from using .extend list method which would add list and tuple likewise.
```suggestion
def install_client(master, client, extra_args=[], user=None,
```

File: ipatests/pytest_ipa/integration/tasks.py [hunk 2]
```diff
@@ -516,14 +516,19 @@ def install_client(master, client, extra_args=(),
...
         '-w', password,
-        '--server', master.hostname] + list(extra_args)
-    )
+        '--server', master.hostname
+    ]
+
+    if unattended:
+        args.append('-U')
+
+    result = client.run_command(args + list(extra_args), stdin_text=stdin_text)
```
Comment (miskopo) [inline ipatests/pytest_ipa/integration/tasks.py]:
```suggestion
    result = client.run_command(args.extend(extra_args), stdin_text=stdin_text)
```

File: ipatests/pytest_ipa/integration/tasks.py (hunk 1 above)
Comment (Tiboris) [reply ipatests/pytest_ipa/integration/tasks.py]: Done

File: ipatests/pytest_ipa/integration/tasks.py (hunk 2 above)
Comment (Tiboris) [reply ipatests/pytest_ipa/integration/tasks.py]: Fixed

**Comment by miskopo** [comment]
Remove tmp commit, otherwise ACK

**Comment by Tiboris** [comment]
master:

* 44bcf0990fd5ed64c9997420b19db161b3b407fe ipatests: Update test tasks for client to be interactive
* 2bc7fb7fd0b0dec3b7fd8d25b4a30aa453537dfd ipatests: Add tests for interactive chronyd config
* d0efb9ea48a96168c73f15a297b3b7da7b6e87e0 ipatests: refactor TestNTPoptions
* 8b7fae30b15d5507da95f91f453b1d816eff6d7b Increase ntp_options test timeout

**Pull request closed without merging**